### DISCUSSION TITLE
"Fantasy Consoles" Discord server down

### DISCUSSION THREAD

**Discussion by scambier** [nesbox/TIC-80#1727] [General]
It seems that following a series of phishing attacks on Discord, the "Fantasy Consoles" server has been taken down.

Are there Discord mods/admins present here that can share the new Discord server address, once it's up again?

**Comment by RobLoach** [comment]
Yes, Seppi is in talks with Discord about hopefully restoring.

**Reply by nesbox** [reply]
The new one is here https://discord.gg/HwZDw7n4dN

**Reply by scambier** [reply]
> Yes, Seppi is in talks with Discord about hopefully restoring.

10 days later, I guess it's definitely dead? :(

Because the quality of the new server is questionable :/

**Reply by RobLoach** [reply]
How is the quality questionable? If you have any suggestions, I'm sure we could bring changes forwards.

Discord support may be questionable. Talk to Seppi.

**Reply by scambier** [reply]
The general chat is unusable because spammed by obnoxious automatic welcome messages. Last time I checked the pinned messages were just random .tic files without any context. Moderation is inexistent (no admin or mod were present to clean some other spammed messages), and the whole thing seems to be administered by literal kids high on memes.

It feels like the opposite of the old server 🤷‍♂️
I guess I'll check back in a few months one things have settled.

**Reply by astrolemonade** [reply]
Is there any news about the fantasy consoles server? It's so sad to see it go!

**Reply by ghost** [reply]
The old server seems to not come back ☹, but there is new not kinda official server Fantasy Consoles 2.0, server seems to be nice done - more fantasy consoles/computers. Some things are missing from the old server, but at least it's not like the new TIC-80 as @scambier wrote previously. Seppi is even on it, but not as an administrator.
Here is invite link https://discord.gg/jcT9CXDgHB
